ng2-material/components/data_table/data_table.ts

```diff
@@ -80,17 +84,6 @@ export class MdDataTable implements AfterContentInit {
       .map((tr: MdDataTableSelectableRow) => tr.selectableValue);
   }
 
-  ngAfterContentInit() {
-    if (this.selectable === true) {
-      if (isPresent(this._masterRow)) {
-        this._masterRow.onChange.subscribe(this.change.bind(this));
-      }
-
-      this._rows.toArray()
-        .map((tr: MdDataTableSelectableRow) => {
-          tr.onChange.subscribe(this.change.bind(this));
-        });
-    }
-  }
+  ngAfterContentInit() {}
```

Comment:
I thought it was for later use as other original components also had empty `ngAfterContentInit`.
I might as well clean it up. :smile: